### PR TITLE
Librarian perform loan extend or return

### LINKED ISSUE #21: Back-office UI:Librarian performs loan return or extend

### PR DESCRIPTION
**Pull request by zzacharo**
closes #21

### CODE REVIEW

File: invenio_app_ils/cli.py [hunk 1]
```diff
@@ -157,6 +156,11 @@ def _get_valid_status(item, items_on_loans):
             "request_expire_date": expire_date.strftime("%Y-%m-%d"),
         }
 
+        if status == "PENDING":
+            loan[Document.pid_field] = "{}".format(item[Document.pid_field])
```
Comment (topless) [inline invenio_app_ils/cli.py:160]: We need the `Document.pid_field` in every case not only when its `PENDING`

File: invenio_app_ils/config.py [hunk 2]
```diff
@@ -50,11 +50,8 @@
 from invenio_circulation.transitions.transitions import (  # isort:skip
     ToItemOnLoan,
     CreatedToPending,
-    ItemInTransitHouseToItemReturned,
-    ItemOnLoanToItemInTransitHouse,
     ItemOnLoanToItemReturned,
-    PendingToItemAtDesk,
-    PendingToItemInTransitPickup,
+    ItemOnLoanToItemOnLoan,
```
Comment (topless) [inline invenio_app_ils/config.py:54]: I understand the state-to-state naming, but I think `ExtendItemOnLoan` would suffice, what happens internally with the states shouldn't bother the names.

**Review by topless — COMMENTED** [review]
Minor comments, can be omitted of course.

File: invenio_app_ils/config.py (hunk 2 above)
Comment (zzacharo) [reply invenio_app_ils/config.py:54]: but this one is on circulation side...create a ticket about that so we can handle it :)

File: invenio_app_ils/cli.py (hunk 1 above)
Comment (zzacharo) [reply invenio_app_ils/cli.py:160]: Yes but right now is not supported in invenio-circulation so to be consistent I din't include it here. I created a ticket about that to handle it https://github.com/inveniosoftware/invenio-circulation/issues/85

Comment (topless) [reply invenio_app_ils/cli.py:160]: 👍 for this one, I am about to address also the issue of the demo loan data and how they reflect reality. We need this one.

File: invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoansSearch/LoansSearch.js [hunk 3]
```diff
@@ -34,21 +33,6 @@ import {
 import './LoansSearch.scss';
 import { default as config } from './config';
 
-const resultsPerPageValues = [
```
Comment (ntarocco) [inline invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoansSearch/LoansSearch.js:37]: Thanks, I missed this one!

**Review by ntarocco — APPROVED** [review]

**Review by kpsherva — APPROVED** [review]

**Pull request merged**